clients/locationClient/locationClient.go

```diff
@@ -55,3 +55,29 @@ func GetLocationList() (LocationList, error) {
 
 	return locationList, nil
 }
+
+func GetLocation(location string) (*Location, error) {
+	if len(location) == 0 {
+		return nil, fmt.Errorf(azure.ParamNotSpecifiedError, "location")
```

Comment:
if we're always formatting this azure.ParamNotSpecifiedError string, it shouldn't probably be exposed. can you please change while you're at it, or maybe in a separate PR?